lib/cmds/space_cmds/alias_cmds/update.js

```diff
@@ -0,0 +1,58 @@
+import * as logging from '../../../utils/log'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { createManagementClient } from '../../../utils/contentful-clients'
+
+export const command = 'update'
+
+export const desc = 'Update an environment alias'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful space alias update --alias-id master --environment-id staging')
+    .option('alias-id', {
+      alias: 'a',
+      describe: 'Id of the alias to create',
+      demandOption: true
+    })
+    .option('environment-id', {
```

Comment:
I wonder if we should call this something else. `target-environment-id`? just because we use --environment-id everywhere else to mean the environment we are acting on?

IDK i'm torn about this

Reply:
That's a valid point